spoken/settings.py

```diff
@@ -338,4 +342,51 @@
         'LOCATION': 'localhost:11211',
         'TIMEOUT': 3600 * 24,
     }
-}
\ No newline at end of file
+}
+
+REST_FRAMEWORK = {
+    'DEFAULT_PERMISSION_CLASSES': (
+        'rest_framework.permissions.IsAuthenticated',
+    ),
+    'DEFAULT_AUTHENTICATION_CLASSES': (
+        'rest_framework_jwt.authentication.JSONWebTokenAuthentication',
+        'rest_framework.authentication.SessionAuthentication',
+        'rest_framework.authentication.BasicAuthentication',
+    ),
+}
```

Comment:
Spoken tutorial has multiple open api end points define in the system which are currently running in production. Adding DEFAULT_PERMISSION_CLASSES in settings file will change the behavior of all api end points. Please handle authentication in individual api end point define by you.

Reply:
You can take help from here. https://www.django-rest-framework.org/api-guide/permissions/

Reply:
As discussed, have moved 'REST_FRAMEWORK' to scriptmanager/settings.py